doc/nrf/gs_ins_mac.rst

```diff
@@ -64,3 +64,38 @@ Install Homebrew and install the required tools using the ``brew`` command line
 .. include:: gs_ins_windows.rst
    :start-after: buildenv_start
    :end-before: buildenv_end
+
+.. _additional_mac_setup:
+
+Setting up executables on macOS
+===============================
+
+If you start SES on macOS by running the file :file:`bin/emStudio`, make sure to complete the following steps:
+
+1. Specify the path to all executables under :guilabel:`Tools` -> :guilabel:`Options` (in the :guilabel:`nRF Connect` tab).
+
+    .. figure:: images/ses_options.png
+         :alt: nRF Connect SDK options in SES on Windows
+
+         nRF Connect SDK options in SES (Windows)
+
+   Use this section to change the SES environment settings later as well.
+
+#. Specify the path to the west tool as additional CMake option, replacing *path_to_west* with the path to the west executable (for example, ``/usr/local/bin/west``):
+
+    .. parsed-literal::
+       :class: highlight
+
+       -DWEST=\ *path_to_west*
+
+
+You can also start SES on macOS from the command line environment.
```

Comment:
I would replace this sentence and the following with:

If you start SES from the command line, it uses the global PATH variable to find the executables, and you do not need to explicitly configure them.